x-pack/plugins/security_solution/server/lib/detection_engine/rules/create_rules.ts

```diff
@@ -76,8 +76,12 @@ export const createRules = async ({
   exceptionsList,
   actions,
   isRuleRegistryEnabled,
-}: CreateRulesOptions): Promise<SanitizedAlert<RuleTypeParams>> => {
-  const rule = await rulesClient.create<RuleTypeParams>({
+  id,
+}: CreateRulesOptions): Promise<SanitizedAlert<AlertTypeParams>> => {
+  const rule = await rulesClient.create<AlertTypeParams>({
+    options: {
+      id,
```

Comment:
With this change to the `createRules` interface to now accept a pre-defined `id` it'll be fine for other consumers to use this to create rules with a specified `id` as well, no? It's not exposed through the route, but just curious as this would be helpful in my work around the rule execution log to be able to test using esarchiver data and work around the event-log RBAC... 🤔

Reply:
It's usable, but I would recommend using `rulesClient.create()` directly in most cases with the newer type specific schemas. I used `createRules()` here because the `AddPrepackagedRulesSchema` is still in the old "every rule type specific field is optional" format and `createRules()` is set up to handle that format. Once we move all the schemas over to more type safe and specific types then we can remove `createRules()` completely.